### PR TITLE
New Talon6 Roll Off Roof driver - v0.1

### PR DESCRIPTION
**Pull request by fenriques**
I finally put my telescope in a hosting facility where Talon6 RoR are in use.
This is a first version of the INDI driver (based on the indi_dome interface).
At this stage I implemented only basic features like open / close / abort and read the status. 
As specified in the dome documentation open / close are implemented using unpark / park method.
So far I'm the only one who tested the driver.

Any feedback welcomed.
Ferrante

### CODE REVIEW

**Comment by knro** [comment]
Great Ferrante! Is this a commercial rolloff solution or just a custom made one?

**Comment by fenriques** [comment]
hi Jasem,
defintely a commercial one: http://observatoriosspag.es/Whats_TALON6.htm
it has been used at e-eye in Spain mainly. But they have 55+ telescopes hosted there (also mine).
The main ASCOM developer Antonio Perez: aperez@fi.upm.es is happy to have an INDI version of the drivers.

File: 3rdparty/indi-talon6/config.h.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+/* Define INDI Data Dir */
+#cmakedefine INDI_DATA_DIR "@INDI_DATA_DIR@"
+
+/* Define Driver version */
+#define INDI_NEXDOME_VERSION_MAJOR @INDI_NEXDOME_VERSION_MAJOR@
```
Comment (knro) [inline 3rdparty/indi-talon6/config.h.cmake]: Needs to be fixed here.

**Comment by knro** [comment]
Great, I can merge whenever you fix one small issue. We would also need documentation for INDI Library. Similar to this: https://indilib.org/devices/domes/nexdome.html but prefreably with screenshots of the various INDI control panels and any information required for end users to operate it propertly. Thank you for this great contribution! we definitely needs more dome drivers.

File: 3rdparty/indi-talon6/config.h.cmake (hunk 1 above)
Comment (fenriques) [reply 3rdparty/indi-talon6/config.h.cmake]: oops...copy and paste

**Comment by fenriques** [comment]
Ok, I edited directly on the online editor as it was a small fix. You can merge it. I will then write on the forum asking for testing on the nightly.

I will write the documentation this weekend. Shall I include it in the build (how??) or send files to you? 

There will be a further version coming in the next days as I'm still missing sync, better state management, mount/weather snooping...

**Comment by knro** [comment]
You can attach a zip file with any documentation information and I'll add it to the INDI site. I can grant you permission as well to manage the page later on.

**Comment by fenriques** [comment]
ok thank you. At this stage it make sense to have short documentation because there are just basic features so I added only a screenshot of main controls because other tabs are not even used...
The page has html tags taken from the nexdome page, hope it helps. while listing details are below.

Listing Details
Website http://observatoriosspag.es/Whats_TALON6.htm
Driver Name Talon6
Driver Executable indi_talon6
Family Domes
Manufacturer O.A.R. MONFRAGÜE
Platforms Linux, BSD, MacOS
Author Ferrante Enriques
Version 0.1

[Talon6.zip](https://github.com/indilib/indi/files/2896771/Talon6.zip)

**Comment by knro** [comment]
Thanks, it's added here: https://indilib.org/devices/domes/talon6-roll-off.html

Can you please send another PR for the Debian package as well? You can just use any existing one debian/indi-* directory as template.

**Pull request merged**